src/main/java/org/broadinstitute/hellbender/tools/ConvertHeaderlessHadoopBamShardToBam.java

```diff
@@ -1,44 +1,78 @@
 package org.broadinstitute.hellbender.tools;
 
-import htsjdk.samtools.*;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SamReader;
+import htsjdk.samtools.SamReaderFactory;
+import htsjdk.samtools.ValidationStringency;
 import org.broadinstitute.barclay.argparser.Argument;
 import org.broadinstitute.barclay.argparser.BetaFeature;
 import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
 import org.broadinstitute.barclay.help.DocumentedFeature;
 import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
-import org.broadinstitute.hellbender.cmdline.programgroups.TestSparkProgramGroup;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.ReadDataProgramGroup;
 import org.broadinstitute.hellbender.engine.spark.datasources.ReadsSparkSink;
 import org.broadinstitute.hellbender.exceptions.UserException;
 import org.broadinstitute.hellbender.utils.read.ReadsWriteFormat;
 import org.broadinstitute.hellbender.utils.spark.SparkUtils;
 
-import java.io.*;
+import java.io.File;
+import java.io.IOException;
 
 
 /**
- * Spark troubleshooting utility that converts a headerless Hadoop bam shard (eg., a part0000, part0001, etc. file
- * produced by {@link ReadsSparkSink}) into a readable bam file by adding a header and a BGZF terminator.
+ * Convert a headerless Hadoop BAM shard into a readable BAM. This is a Spark troubleshooting utility that converts a
+ * headerless Hadoop BAM shard (eg., a part-r-00000, part-r-00001, etc. file
```

Comment:
ReadsSparkSink is an implementation detail, so I think we can exclude it from the doc. Instead, I've added an explanation that a sharded BAM is produced by any Spark tool invoked with --sharded-output set to true. Also removed the reference to Hadoop since it is redundant - as far as I know, all shards are Hadoop in GATK.

As a side note, I noticed that this flag is still in camelCase (--shardedOutput). I'm assuming this will get converted to kebab.